test/test_extractor.py

```diff
@@ -70,7 +70,6 @@ def extension_list(self) -> list[str]:
         return []
 
     @pytest.mark.asyncio
-    @pytest.mark.skipif(sys.version_info < (3, 8), reason="py3.7 fails sometimes")
```

Comment:
There's a bit more Python 3.7 specific code in this file, you can find it all by searching `3, 8`.